src/pages/BookAppointment/BookAppointment.scss

```diff
@@ -262,4 +315,93 @@
       display: none;
     }
   }
+
+  .book-appointment--dark {
+
+    .headerForm-container {
+      background-color: $clr-secondary-dk;
+      width: 100%;
+      height: 100vh;
+    }
+
+    .header {
+      box-shadow: none;
```

Comment:
There needs to be a shadow under the header on dark mode. Take a look at the Staff page and the settings page for the SCSS.